### ISSUE TITLE
Can people get the result in the paper if they train it from 'init'?

### ISSUE THREAD

**Issue by johnnylu305** [zbf1991/RRM#10]
Hi, nice to meet you.
There are two pretrain weights about your classification network.
One is "ilsvrc-cls_rna-a1_cls1000_ep-0001.params" and another one is "res38_cls.pth".
My question is that can people get the same result in your paper if they train from "ilsvrc-cls_rna-a1_cls1000_ep-0001.params"?

**Comment by zbf1991** [comment]
yes, no matter train from 'init' or from 'cls', the final performance is the same. I have tested on my computer. A few days ago, another researcher told me he/she cannot get the result using the 'init', I have found some bugs and fixed it. If you cannot get the result, please let me know, I will check it again.

**Comment by johnnylu305** [comment]
@zbf1991 Thank you for your explanation. Sorry, I forgot to ask another minor question.
Why you set densecrfloss to 1e-7? Doesn't it disable your energy loss?

**Comment by johnnylu305** [comment]
I think densecrfloss is just a hyperparameter to adjust energy loss.
This is because the value of energy loss is very small(ex: -4700377).
